Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/Approval/DataStanadard_400_ApprovalTests.Verify.Extensions.SampleStudentTranscript.1.0.0_Std_4.0.0_Models_Entities_Entities.generated.approved.cs

```diff
@@ -157,7 +154,7 @@ public override bool Equals(object obj)
             {
                 if (entry.Value is string)
                 {
-                    if (!_databaseEngineSpecificStringComparer.Equals((string) entry.Value,(string) thoseKeys[entry.Key]))
+                    if (!GeneratedArtifactStaticDependencies.DatabaseEngineSpecificStringComparer.Equals((string) entry.Value, (string) thoseKeys[entry.Key]))
```

Comment:
These changes are needed for Approval tests to pass again, but they are unrelated to this ticket (appear to bleeding in from changes made in ODS-5907). Look for the `[Required]` attribute to be applied to members generated for items with updated cardinality in the model files.